app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -168,7 +168,7 @@ class ProfileChooserFragmentPresenter @Inject constructor(
   ) {
     binding.viewModel = model
     binding.presenter = this
-    binding.root.setOnClickListener {
+    binding.root.profile_item.setOnClickListener {
```

Comment:
Prefer accessing the view directly instead of using the synthetic import, e.g. ``profileItem``

Reply:
Done, resolving on behalf of ben